src/actions/event/event-list-action.js

```diff
@@ -38,9 +38,10 @@ export function getEvents(filters) {
       dispatch(setErrorAllertFromResponse(response));
       return Promise.reject();
     }
+    // TODO: MOCKUP
     const jsonRes = await response.json();
-    jsonRes.items = eventMockup;
-    dispatch(getEventsData(jsonRes));
+
+    dispatch(getEventsData({ jsonRes, items: eventMockup }));
```

Comment:
Are you sure, that this method receive `jsonRes` and `items` params?

Reply:
Ooopss... I forgot to write the spread operator